bayesian_asimetrix/statistics.py

```diff
@@ -70,3 +68,46 @@ def prob_density_estimate(df_post: pd.DataFrame) -> np.ndarray:
     dens = kde.pdf(df_post.values.T)
 
     return dens
+
+def summary_table(df_post: pd.DataFrame, alpha: float=0.05, round_to: int=2) -> pd.DataFrame:
+    """
+    Summarizes the information of the posteriori distribution of the model parameters.
+    For more information about the ESS and Rhat diagnostics, please refet to:
+    https://mc-stan.org/rstan/reference/Rhat.html
+    Args:
+        df_post (pd.DataFrame):  A posteriori joint distribution of the model parameters
+        alpha (float=0.05):  Significance for the High Density Interval. A number between 0 and 1.
+        round_to (int=2): Number of decimals used to round results. Defaults to 2. Use None to return the raw numbers
+    Returns:
+        pd.DataFrame: Summary Table
+        - Mode
+        - High Density Interval
+        - Bulk Effective Sample Size
+        - Tail Effective Sample Size
+        - Rhat
+    """
+
+    # Summmary Table Initialization
+    df_summary = pd.DataFrame(index=df_post.columns)
+
+    # Modes
+    df_summary['Mode'] = df_post.apply(mode_estimate, raw=True)
+
+    # High Density Intervals
+    df_hdi = df_post.apply(lambda x: high_density_interval(x.values, alpha))
+    df_hdi.index = [f'HDI_{alpha/2*100}%', f'HDI_{100*(1 - alpha/2)}%']
+    df_hdi = df_hdi.transpose()
+    df_summary = pd.concat([df_summary, df_hdi], axis=1)
+
+    # Effective Sample Size
+    df_summary['Bulk ESS'] = df_post.apply(az.ess, raw=True, **{'method': 'bulk'})
+    df_summary['Tail ESS'] = df_post.apply(az.ess, raw=True, **{'method': 'tail'})
```

Comment:
Si es solo un atributo **{} parece un overkill. Y de hecho si es más de un atributo pues asígnelo a un diccionario  en una parte de configuración y después lo llama acá.
```suggestion
    df_summary['Bulk ESS'] = df_post.apply(az.ess, raw=True, method='bulk')
    df_summary['Tail ESS'] = df_post.apply(az.ess, raw=True, method='tail')
```